config/application.rb

```diff
@@ -13,9 +13,6 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
 
-    # Default app time zone
-    config.time_zone = 'Madrid'
```

Comment:
why not comment it and add a comment like:
"""
Can not force time_zone as all timestamps in DB are in UTC and ought to be updated if this config was applied.
"""

Reply:
@tramuntanal changes applied.